metricbeat/module/mysql/overview/manifest.yml

```diff
@@ -0,0 +1,25 @@
+default: true
+input:
+  module: mysql
+  metricset: query
+  defaults:
+    sql_response_format: variable
+    namespace: overview
+    sql_queries:
+      - query: >
+          SELECT *
```

Comment:
🤔  I'm a little bit confused :) , is this a light module? Cause I see the same query in implementation in `metricbeat/module/mysql/overview/overview.go`.

Reply:
Yeah, I initially created it as a "normal" module and then I realized that I could try to invest some time in doing it as a lightweight module. I forgot completely about this file. I have just deleted it.
Thanks for the heads up :smiley:

Reply:
👍